client/src/components/registration/components/authentication/AuthenticatedRoute.js

```diff
@@ -0,0 +1,25 @@
+import React from "react";
```

Comment:
take a look at the folder structure for this file - components/registration/components ... looks like folders have been duplicated in the registration folder